website/routes.py

```diff
@@ -1150,6 +1156,11 @@ def make_url_map(app):
         ], 'get', project_views.register.node_register_template_page, json_renderer),
 
         Rule([
+            '/project/<pid>/retract_registration/',
+            '/project/<pid>/node/<nid>/retract_registration/'
+        ], 'post', project_views.register.node_registration_retraction, json_renderer),
```

Comment:
Would it make sense to have two separate view for GET vs. POST? That avoids having to do `if request.method == 'POST'`. Avoiding LocalProxies is almost always +1 in my book.